source/_docs/installation/windows.markdown

```diff
@@ -39,3 +41,23 @@ C:\...\> pip3 install --upgrade homeassistant
 ```
 
 Then start Home Assistant.
+
+## {% linkable_title Installing in a python virtual environment %}
+Open CMD.exe and paste the folowing commands (one at the time):
+```bash
+C:\...\> py -m venv homeassistant
+C:\...\> cd homeassistant
+C:\...\> Scripts\activate.bat
+C:\...\> py -m pip install wheel setuptools pip
+C:\...\> py -m pip install homeassistant
+C:\...\> hass --open-ui
```

Comment:
`--open-ui` is only useful if you want to browse the Home Assistant frontend the local machine. Should be removed here.